scripts/test_complete_workflow.py

```
@@ -0,0 +1,310 @@
+"""완전한 워크플로우 테스트 스크립트"""
+
+import asyncio
+import aiohttp
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)